sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/RecognizeOptions.java

```diff
@@ -3,64 +3,19 @@
 package com.azure.ai.formrecognizer.models;
 
 import com.azure.core.annotation.Fluent;
-import reactor.core.publisher.Flux;
 
-import java.io.InputStream;
-import java.nio.ByteBuffer;
 import java.time.Duration;
 
 /**
  * Options that may be passed when using recognize APIs on Form Recognizer client.
  */
 @Fluent
 public class RecognizeOptions {
-    private final InputStream form;
-    private final Flux<ByteBuffer> formData;
-    private final long length;
-    private final String formUrl;
     private FormContentType formContentType;
```

Comment:
In .NET we call this field `ContentType` only (without the `Form` prefix). Are we concerned about consistency in fields names? If `formContentType` feels more idiomatic to Java, I believe that's not a big concern on the .NET side and can be easily updated.

Reply:
I think we agreed to keep this model named `FormContentType`. 
@kristapratico and @jeremymeng can you confirm what Python and JS have?

Reply:
The model is called `FormContentType` as well in .NET. I'm talking about the `formContentType`, which is simply `ContentType` in .NET. I know it's private, but since you have the public `getFormContentType` method, I was wondering whether we should update its name in .NET.

Reply:
We renamed the type to `FormContentType` to differentiate from general content type as only a handful content types are supported.

Reply:
We are all in the same page with the name of the type.
I think what @kinelski is pointing out is the name of the property. Currently .NET, JS, and Python have `Content Type` for the property name.

Reply:
we don't have this in an option bag in JS.  for parameters it's just `contentType`

Reply:
I am inclined to keep it consistent with the property name unless we have any strong objections.

Reply:
I am not against it, although I will like to understand why the deviation from the other languages though

Reply:
I guess, Js and Python don't have this issue, just .NET and Java, if you are not too against it, could rename it to `formContentType`?